doc/source/getting_started/comparison/includes/copies.rst

```diff
@@ -11,3 +11,17 @@ or overwrite the original one:
    .. code-block:: python
 
       df = df.sort_values("col1")
+
+.. note::
+
+   You will see an ``inplace=True`` or ``copy=False`` keyword argument available for
+   some methods:
+
+   .. code-block:: python
+
+      df.replace(5, inplace=True)
```

Comment:
I don't think replace is a good example. We are keeping ``inplace`` there.

Reply:
That’s why I am using it. I could add an explicit comment that replace is one of the few methods where we are keeping it

Reply:
Thanks, it might be good to also give an example of something that would lose inplace, for comparison's sake.

Reply:
I don't think we would need to go to in detail, just say e.g.

"Some methods cannot operate inplace (e.g. ``reindex``), or would already avoid copying the data with Copy on Write. (e.g. ``rename``) 

It is recommended that the ``inplace``/``copy`` parameters be avoided for those methods".

No big deal if we don't add this, though, since people can always read the PDEP.

Reply:
Ah got you, I thought you wanted to add a code example.

Reply:
I'll leave the suggestion to not use them out. When used in the appropriate context it _can_ speed up your code. I think it's clear enough that users can figure the consequences out for themself.